### PR TITLE
IE11 Auth Code Compatibility

### PR DESCRIPTION
**Pull request by tnorling**

### CODE REVIEW

File: lib/msal-common/src/utils/StringUtils.ts [hunk 1]
```diff
@@ -41,6 +41,10 @@ export class StringUtils {
         return (typeof str === "undefined" || !str || 0 === str.length);
     }
 
+    static startsWith(str: string, search: string): boolean {
+        return str.substring(0, search.length -1) === search;
+    }
+
```
Comment (pkanher617) [inline lib/msal-common/src/utils/StringUtils.ts:47]: Tests for this function?

File: lib/msal-common/src/utils/StringUtils.ts [hunk 2]
```diff
@@ -41,6 +41,10 @@ export class StringUtils {
         return (typeof str === "undefined" || !str || 0 === str.length);
     }
 
+    static startsWith(str: string, search: string): boolean {
+        return str.substring(0, search.length -1) === search;
```
Comment (jasonnutter) [inline lib/msal-common/src/utils/StringUtils.ts:45]: I would use `indexOf` and `length - 1`

File: lib/msal-common/src/url/UrlString.ts [hunk 3]
```diff
@@ -39,7 +39,7 @@ export class UrlString {
             url = url.toLowerCase();
         }
 
-        if (url && !url.endsWith("/")) {
+        if (url && (url.charAt(url.length - 1) !== "/")) {
```
Comment (jasonnutter) [inline lib/msal-common/src/url/UrlString.ts:42]: Can we add an `endsWith` string util similar to startsWith?

Comment (tnorling) [reply lib/msal-common/src/url/UrlString.ts]: Done

File: lib/msal-common/src/utils/StringUtils.ts (hunk 2 above)
Comment (tnorling) [reply lib/msal-common/src/utils/StringUtils.ts]: Done

File: lib/msal-common/src/utils/StringUtils.ts (hunk 1 above)
Comment (tnorling) [reply lib/msal-common/src/utils/StringUtils.ts]: Done

**Pull request merged**